packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -293,36 +293,145 @@ describe('packages/date-picker', () => {
       });
 
       describe('Chevrons', () => {
-        test('Left is disabled if prev. month is entirely out of range', async () => {
-          const { openMenu } = renderDatePicker({
-            min: new Date(Date.UTC(2023, Month.December, 1)),
+        describe('left', () => {
+          describe('is disabled', () => {
+            test('when the year and month is before the min', async () => {
+              const { openMenu } = renderDatePicker({
+                min: new Date(Date.UTC(2023, Month.December, 1)),
+                value: new Date(Date.UTC(2022, Month.December, 1)),
+              });
+
+              const { leftChevron } = await openMenu();
+              expect(leftChevron).toHaveAttribute('aria-disabled', 'true');
+            });
+            test('when the year and month is the same as the min', async () => {
+              const { openMenu } = renderDatePicker({
+                min: new Date(Date.UTC(2023, Month.December, 10)),
+                value: new Date(Date.UTC(2023, Month.December, 1)),
+              });
+
+              const { leftChevron } = await openMenu();
+              expect(leftChevron).toHaveAttribute('aria-disabled', 'true');
+            });
+            test('when the year is the same as the min and the month is before the min', async () => {
```

Comment:
I think this test case would be covered by the first one (L298), (unless there is logic that makes us think that this could be brittle)